get_regression_metrics should accept lists

```/workspaces/modelling-tools/fhdw/tests/test_tracking.py::test_log_metrics_to_mlflow failed: fix_data = ([1.0, 2.0, 3.0, 4.0], [1.1, 2.2, 2.8, 4.2])

    def test_log_metrics_to_mlflow(fix_data):
        """Test case for log_metrics_to_mlflow function."""
        y_true, y_pred = fix_data
        mock_log_metrics = MagicMock()
    
        # Patch mlflow.log_metrics with mock object
        with patch("mlflow.log_metrics", mock_log_metrics):
>           log_metrics_to_mlflow(y_true, y_pred)

fhdw/tests/test_tracking.py:27: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
fhdw/modelling/tracking.py:12: in log_metrics_to_mlflow
    metrics = get_regression_metrics(y_true=y_true, y_pred=y_pred)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

y_true = [1.0, 2.0, 3.0, 4.0], y_pred = [1.1, 2.2, 2.8, 4.2]

    def get_regression_metrics(y_true, y_pred):
        """Get dictionary of common regression metrics.
    
        Args:
            y_true: The actual values of the ground truth.
    
            y_pred: The inference values made by the model.
        """
>       if (y_true < 0).any() or (y_pred < 0).any():
E       TypeError: '<' not supported between instances of 'list' and 'int'

fhdw/modelling/evaluation.py:23: TypeError```

--- a/fhdw/modelling/evaluation.py
+++ b/fhdw/modelling/evaluation.py
@@ -20,14 +20,15 @@ def get_regression_metrics(y_true, y_pred):
 
         y_pred: The inference values made by the model.
     """
-    if (y_true < 0).any() or (y_pred < 0).any():
+    try:
+        rmsle = mean_squared_log_error(y_true=y_true, y_pred=y_pred, squared=False)
+    except ValueError:
+        rmsle = None
+
         warnings.warn(
             "Mean Squared Logarithmic Error cannot be used when "
             "targets contain negative values. Therefore it is set to None here."
         )
-        rmsle = None
-    else:
-        rmsle = mean_squared_log_error(y_true=y_true, y_pred=y_pred, squared=False)
 
     metrics = {
         "MAE": mean_absolute_error(y_true=y_true, y_pred=y_pred),

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -4,7 +4,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "fhdw-modelling"
-version = "5.1.1"
+version = "5.1.2"
 description = "A collection of modelling tools. Intended for generic usage in FHDW projects."
 authors = ["Pascal Niggemeier <pascal.niggemeier@fhdw.de>"]
 license = "MPL-2.0"

